app/components/search_result/search_result.html.erb

```diff
@@ -3,6 +3,8 @@
     <%= c 'contributor_row', contributor: result %>
   <% when 'Message' %>
     <%= c 'excerpt', title: result.sender_name, text: result.text, link: result.chat_message_link, date: result.created_at %>
+  <% when 'Request' %>
+    <%= c 'excerpt', title: result.title, text: result.text, link: request_path(result), date: result.created_at %>
```

Comment:
```suggestion
    <%= c 'excerpt', title: result.title, text: result.text, link: request_path(result), date: result.broadcasted_at || result.created_at %>
```

Should we have some UI that distinguished planned/scheduled requests from broadcasted ones?